src/App.tsx

```diff
@@ -34,62 +35,114 @@ export function getReorderedGoods(
   // To avoid the original array mutation
   const visibleGoods = [...goods];
 
+  if (isReversed) {
+    visibleGoods.reverse();
+  }
```

Comment:
your reverse doesn't work correctly - it works only before sorting - check it
what is expected - I press sort by - then reverse is reversing sorted list